### DISCUSSION TITLE
Where to correct typo in docs?

### DISCUSSION THREAD

**Discussion by fiala-sns** [jdhitsolutions/PSScriptTools#132] [General]
I am wondering where to correct typo in docs? CONTRIBUTING.md says not to create PR on MD files.
Sorry if it is a stupid question, I am new to Git/Github and all this stuff.

**Comment by jdhitsolutions** [comment]
You can submit a PR on the MD files. I didn't want people to push a PR with a new help XML file.  Or, if it is a simple fix, post it here and I'll update the doc.

**Reply by fiala-sns** [reply]
Thank you for your prompt reply. I pushed it.
